controllers/imagejob/imagejob_controller.go

```diff
@@ -365,11 +388,34 @@ func (r *Reconciler) handleNewJob(ctx context.Context, imageJob *eraserv1.ImageJ
 			return err
 		}
 		log.Info("Started "+containerName+" pod on node", "nodeName", nodeName)
+
+		// send pod to podQueue to create thread and wait for pod ready phase
+		waitGroup.Add(1)
+		podQueue <- pod
```

Comment:
I'm thinking it's probably better to send a non-pointer `types.NamespacedName` with the pod's namespace and name, over the channel. We're not using any other info from the pod, so it's better to send less if we can. Generally speaking, sending pointers across threads isn't a great idea: a thread can end up with a dangling pointer.